src/iso19111/c_api.cpp

```diff
@@ -1125,8 +1155,7 @@ PJ_TYPE proj_get_type(const PJ *obj) {
  * @return TRUE if it is deprecated, FALSE otherwise
  */
 int proj_is_deprecated(const PJ *obj) {
-    assert(obj);
-    if (!obj->iso_obj) {
+    if (!obj || !obj->iso_obj) {
```

Comment:
I didn't raise an error here since it didn't pass a context through. There are a couple like this one.